src/sctools/barcode.py

```diff
@@ -277,7 +277,7 @@ def __init__(self, errors_to_barcodes: Mapping[str, str]):
         if not isinstance(errors_to_barcodes, Mapping):
             raise TypeError(
                 f'The argument "errors_to_barcodes" must be a mapping of erroneous barcodes to correct '
```

Comment:
Why isn't this changed?

Reply:
Weird, I was originally thinking since the single-quote -> double-quote conversion in this PR was done by `black`, which seems to intend to leave `f-string`s since they are indeed not strings but expressions. But I'm not sure as we see line 280 was converted ... Might be a bug of black and we could manually convert this...